### PR TITLE
Prompt toolkit 3.0 support

### PR DESCRIPTION
**Pull request by qlixed**
Format files using Black, adjusting the flake8 params to set the format as Black default. I would probably recommend to set black only for formating to avoid collision with flake8 as there are different defaults.
I try the base examples and so far all works ok, the only one that don't work for my is asyncssh, but I'll check that later.
With all this changes the travis test should pass all.

### CODE REVIEW

**Comment by qlixed** [comment]
Hey @jonathanslenders just bumping this for your attention. Thanks!

**Comment by qlixed** [comment]
Or maybe @BoboTiG can help with this?

File: ptterm/backends/posix_utils.py [hunk 1]
```diff
@@ -30,9 +30,11 @@ def pty_make_controlling_tty(tty_fd):
...
             os.close(fd)
     # which exception, shouldnt' we catch explicitly .. ?
-    except:
-        # Already disconnected. This happens if running inside cron.
-        pass
+    except Exception as e:
+        # Already disconnected. This happens if running inside cron
+        # If we're disconnected I understand that I can discard the exception
+        del e
+        # pass
```
Comment (BoboTiG) [inline ptterm/backends/posix_utils.py:37]: I would prefer using that pattern instead:
```python
except Exception:
    # Already disconnected. This happens if running inside cron.
    pass
```

File: ptterm/backends/darwin.py [hunk 2]
```diff
@@ -63,7 +63,8 @@ def get_proc_info(pid):
     # Request the process data.
     reslen = oldlen.value
     old = (c_ubyte * reslen)()
-    oldp = pointer(old)
+    # Appears to not be used:
+    # oldp = pointer(old)
```
Comment (BoboTiG) [inline ptterm/backends/darwin.py:67]: You could just remove the line IMO.

File: ptterm/backends/posix_utils.py [hunk 3]
```diff
@@ -47,9 +49,12 @@ def pty_make_controlling_tty(tty_fd):
...
             )
     # which exception, shouldnt' we catch explicitly .. ?
-    except:
+    # Don't know what exception
+    except Exception as e:
         # Good! We are disconnected from a controlling tty.
-        pass
+        # If we're disconnected I understand that I can discard the exception
+        del e
+        # pass
```
Comment (BoboTiG) [inline ptterm/backends/posix_utils.py:57]: Same remark here.

**Review by BoboTiG — COMMENTED** [review]
Thanks @qlixed. I reviewed from a high perspective as I am not using the module, nor I tested those changes.

**Pull request closed without merging**